test/end-to-end/core.sh

```diff
@@ -180,6 +180,7 @@ wait_for_command "oc get rc/failing-dc-1 --template={{.metadata.annotations}} |
 oc logs dc/failing-dc | grep 'test pre hook executed'
 oc deploy failing-dc --latest
 oc logs --version=1 dc/failing-dc
+oc logs --previous dc/failing-dc
```

Comment:
yes, see L180. I will update to grep the pre hook log that we inline in failing-dc